requirements.txt

```diff
@@ -1,18 +1,46 @@
-wheel>=0.29
-pandas>=0.24.2
-numpy>=1.18.4
-dash==1.16.2
-dash-bootstrap-components==0.10.6
-dash-leaflet==0.1.4
-dash-extensions==0.0.51
-gunicorn>=20.0.4
-Flask-Caching==1.3.3
-Flask-Sockets==0.2.1
-git+https://github.com/pyronear/pyro-api.git#subdirectory=client
-python-dotenv>=0.15.0
-geopy>=2.1.0
-requests>=2.25.0
-pytz>=2020.1
-werkzeug==0.16.0
-MarkupSafe==2.0.1
-protobuf==3.14.0
+blinker==1.4
+brotli==1.0.9; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.3.0"
+cachelib==0.9.0; python_version >= "3.7"
+certifi==2022.6.15; python_version >= "3.7" and python_version < "4"
+charset-normalizer==2.1.0; python_version >= "3.7" and python_version < "4" and python_full_version >= "3.6.0"
+click==8.1.3; python_version >= "3.7" and python_full_version < "3.0.0" or python_full_version >= "3.3.0" and python_version >= "3.7"
+colorama==0.4.5; python_version >= "3.7" and python_full_version < "3.0.0" and platform_system == "Windows" or python_full_version >= "3.5.0" and python_version >= "3.7" and platform_system == "Windows"
+dash-bootstrap-components==0.13.1; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.5.0" and python_version < "4")
+dash-core-components==1.17.1; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version < "4"
+dash-extensions==0.0.71
+dash-html-components==1.1.4; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version < "4"
+dash-leaflet==0.1.23
+dash-table==4.12.0; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version < "4"
```

Comment:
Thanks for update dash static versions. Will run some additional tests to make sure that everything is working as expected !

Reply:
This file is generated from the specifiers I put in `pyproject.toml`. So the process now is to :
- edit pyproject.toml
- it will generate poetry.lock & requirements.txt